# HG changeset patch
# User Benjamin Lindner <lindnerb@users.sourceforge.net>
# Date 1233341830 18000
#      Fri Jan 30 13:57:10 2009 -0500
# Node ID 016e14be00c66ddd9af48a18181cd22ef6713c47
# Parent  2264092e9f394e6c01eeef2def517c6c8ec7091d
missing semicolon in display.cc

diff --git a/src/display.cc b/src/display.cc
--- a/src/display.cc
+++ b/src/display.cc
@@ -43,17 +43,17 @@ void
 display_info::init (void)
 {
 #if defined (OCTAVE_USE_WINDOWS_API)
 
   HDC hdc = GetDC (0);
 
   if (hdc)
     {
-      dp = GetDeviceCaps (hdc, BITSPIXEL)
+      dp = GetDeviceCaps (hdc, BITSPIXEL);
 
       ht = GetDeviceCaps (hdc, VERTRES);
       wd = GetDeviceCaps (hdc, HORZRES);
 
       double ht_mm = GetDeviceCaps (hdc, VERTSIZE);
       double wd_mm = GetDeviceCaps (hdc, HORZSIZE);
 
       rx = wd * 25.4 / wd_mm;
